app/schemas/care_plan.py

```diff
@@ -11,7 +11,18 @@ class Config:
 
 class CarePlanCreate(CarePlan):
     date: Optional[datetime.datetime]
+    progress_date: Optional[str]
```

Comment:
Why are all dates are str not datetime?

Reply:
this is for the convenience of interacting with the front, but where I write to the database there is of course type datetime